djongo/sql2mongo/query.py

```diff
@@ -429,12 +429,22 @@ def execute(self):
                 ins[fld] = v
             docs.append(ins)
 
-        res = self.db_ref[self.left_table].insert_many(docs, ordered=False)
+        if len(docs) == 1:
```

Comment:
Why exactly is this a bug? pymongo lets you insert one document in insert_many.

Reply:
I don't recall the exact error, but sometimes the insert fail. This fix ensure that the bug will not occurs again